### PR TITLE
Fixed blocked exit of CLI program due to REST ready channel

### PR DESCRIPTION
**Pull request by oxisto**
The `ready` channel in the `rest` package was blocking the regular program exit.

### CODE REVIEW

File: rest/rest.go [hunk 1]
```diff
@@ -193,6 +186,22 @@ func RunServer(ctx context.Context, grpcPort int, httpPort int, corsOpts ...CORS
 	return srv.Serve(sock)
 }
 
+// StopServer is in charge of stopping the REST API.
+func StopServer(ctx context.Context) {
```
Comment (lebogg) [inline rest/rest.go:190]: Exported on purpose?

**Review by lebogg — APPROVED** [review]
No blocking on my local machine anymore.

File: rest/rest.go (hunk 1 above)
Comment (oxisto) [reply rest/rest.go:190]: Kind of. Since RunServer was exported, I also exported StopServer.

Comment (lebogg) [reply rest/rest.go:190]: Ok. Thus, engine could stop REST server manually. Sounds reasonable

**Pull request merged**